lib/ansible/cli/galaxy.py

```diff
@@ -897,7 +901,8 @@ def execute_verify(self):
 
         resolved_paths = [validate_collection_path(GalaxyCLI._resolve_path(path)) for path in search_paths]
 
-        verify_collections(requirements, resolved_paths, self.api_servers, (not ignore_certs), ignore_errors)
+        verify_collections(requirements, resolved_paths, self.api_servers, (not ignore_certs), ignore_errors,
+                           allow_pre_release=context.CLIARGS['pre_release'])
```

Comment:
Should this just always work? I can see why installation should have the flag, but I'm not sure if there's a benefit to not allowing pre-releases to be verified by default.

Reply:
Done